src/canvas/DesktopElement.ts

```diff
@@ -400,6 +416,19 @@ export default class GmfDesktopCanvas extends BaseElement {
     this.subscriptions.push(
       panels.getActiveFooterPanel().subscribe({
         next: (panel: string) => {
+          let height = '0';
+          if (panel) {
+            const styles = getComputedStyle(document.documentElement);
+            height = styles.getPropertyValue(`--footer-panel-height-${panel}`);
+            if (!height) {
+              height = styles.getPropertyValue(`--footer-panel-height`);
+            }
+            if (!height) {
```

Comment:
same condition as line 423 ?

Reply:
Yes, but between we have a `height = ...`